src/components/AllProducts.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import {
+	getCurrentUser,
+	storeCurrentUser,
+	getCurrentToken,
+	clearCurrentUser,
+} from '../auth';
+import { Product } from './Product.js';
+
+const AllProducts = ({ products }) => {
+	const [products, setProducts] = useState([]);
+	const handleProducts = async () => {
```

Comment:
perhaps this should be a useEffect to load the products when the component mounts?